### DISCUSSION TITLE
Extension to disable file preview while browsing find in files search results

### DISCUSSION THREAD

**Discussion by opsidjflksdf** [microsoft/vscode-discussions#1182] [Extension Development QnA] [answered]
Hello,
I can't seem to find a setting to disable the previewing of files (them being open and cursor moved to the location of the line number), when navigating with the Up and Down arrow keys in the find-in-files search results. 
I was looking at writing an extension to somehow do that, but I don't really see a solution other than duplicating the whole search-results functionality and removing the preview on change focus. Would there be an easier way? I would appreciate any pointers.
Thanks!

**Comment by isidorn** [comment, marked as answer]
Sorry but not possible to do from an extension.

**Reply by opsidjflksdf** [reply]
Thanks for your reply! If an extension for this is not possible, would there be any way to achieve this in VSCode?

**Reply by isidorn** [reply]
Not using the extension API. 
VS Code could do it with a setting - so feel free to file a new feature request in the vscode repo.
